src/client/texturesource.cpp

```diff
@@ -340,24 +298,22 @@ u32 TextureSource::generateTexture(const std::string &name)
 	std::set<std::string> source_image_names;
 	video::IImage *img = m_imagegen.generateImage(name, source_image_names);
 
-	video::ITexture *tex = NULL;
+	video::ITexture *tex = nullptr;
 
-	if (img != NULL) {
+	if (img != nullptr) {
```

Comment:
null checks of any kind can always be changed to `var` or `!var`

Reply:
Fixed.